chart/env/common.go

```diff
@@ -44,17 +43,13 @@ var (
 
 func init() {
 	circleCI = os.Getenv(EnvVarCircleCI)
+	circleSHA = os.Getenv(EnvVarCircleSHA)
 	keepResources = os.Getenv(EnvVarKeepResources)
 
 	kubeconfig = os.Getenv(EnvVarE2EKubeconfig)
 	if kubeconfig == "" {
 		kubeconfig = e2eHarnessDefaultKubeconfig
 	}
-
-	circleSHA = os.Getenv(EnvVarCircleSHA)
-	if circleSHA == "" {
```

Comment:
Why is it empty/undefined in e2etests job? https://circleci.com/gh/giantswarm/e2etests/225

Reply:
Not sure. I checked the circle config and the build and the env var is populated.

But it will fail when running locally so I don't like that it panics.

Reply:
Once merged, will rebase kubernetes-1.15.4 branch of e2esetup, so e2etests and and other projects depending on e2esetup can be updated.

Reply:
@sslavic Thanks. Done.

Reply:
@rossf7 `e2esetup` `kubernetes-1.15.4` branch has been rebased on top of latest master.

Reply:
Thanks build is fixed. https://github.com/giantswarm/e2etests/pull/52

Reply:
Maybe it is not passed to docker container?